ui-tests/cypress/support/pageObjects/content/assets/AddPage.js

```diff
@@ -0,0 +1,45 @@
+import {htmlElements} from '../../WebElement.js';
+
+import AdminContent from '../../app/AdminContent';
+import AssetsPage   from './AssetsPage';
+import AdminPage    from '../../app/AdminPage';
+
+
+export default class AddPage extends AdminContent {
+
+  addButton     = `${htmlElements.input}#submit`;
+  fileInput     = `${htmlElements.input}[type=file]`;
+  groupSelector = `${htmlElements.select}[id="mainGroup"]`;
+
+  static openPage(button) {
+    cy.assetsAdminConsoleController().then(controller => controller.intercept({method: 'GET'}, 'addPageLoadingGet', '/new.action?*'));
+    cy.get(button).click();
+    cy.wait('@addPageLoadingGet');
+  }
+
+  getFileInput() {
+    return this.get()
+               .find(this.fileInput);
+  }
+
+  selectFiles(...fileName) {
+    this.getFileInput().eq(0).selectFile(fileName, {force: true});
```

Comment:
This method is not returning anything; since it's an _operation_ method, it should return the current page.
Moreover, we have a _select_ method on the `Content` class; is it needed to use the `force: true`?

Reply:
I did not investigate this point. I simply adapted the method that already existed and only checked how it worked. 
I'll work on the necessity of {force: true}  for this method right away. :)